config.py

```diff
@@ -0,0 +1,12 @@
+import argparse
+
+def arg_parse():
+    parser = argparse.ArgumentParser(description='PyTorch DQN Training')
+
+    parser.add_argument('--savePrefix', type=str, help='the save directory of the running results')
+    parser.add_argument('--epsStart', type=float, help='the initial epsilon')
```

Comment:
How about adding `epsEnd` and `epsDecay` arguments as well (since they always take effect together)?